### PR TITLE
13600 Https by default

### PR DESCRIPTION
**Pull request by Sergio-Mira**
https://swrvedev.jira.com/browse/SWRVE-13600

### CODE REVIEW

File: unity3d/Assets/Swrve/SwrveSDK/SwrveConfig.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#if UNITY_IPHONE && !UNITY_5_0_0
+#if !UNITY_5_0_0
 #define SWRVE_USE_HTTPS_DEFAULTS
 #endif
```
Comment (seaders) [inline unity3d/Assets/Swrve/SwrveSDK/SwrveConfig.cs:3]: If not Unity_5_0_0 check?  Is this still correct?

Comment (Sergio-Mira) [inline unity3d/Assets/Swrve/SwrveSDK/SwrveConfig.cs:3]: Hmm, I might turn that into !(UNITY_IPHONE && UNITY_5_0_0) as there is an iOS issue where https just doesn't work on 5.0.0

Comment (seaders) [inline unity3d/Assets/Swrve/SwrveSDK/SwrveConfig.cs:3]: But we just gots to fix that rather than ignore / avoid it.

Comment (Sergio-Mira) [inline unity3d/Assets/Swrve/SwrveSDK/SwrveConfig.cs:3]: It is in the generated code, no way around it for iOS 5.0.0

**Pull request merged**